media-libs/pulseaudio-qt6/pulseaudio-qt6-9999.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+ECM_HANDBOOK="forceoptional"
+ECM_QTHELP="true"
+ECM_TEST="optional"
+KFMIN=5.245.0
+QTMIN=6.6.0
+KDE_ORG_NAME="${PN/qt6/qt}"
+MY_PN="${PN/qt6/qt}"
+MY_P="${P/qt6/qt}"
+
+inherit ecm kde.org
+
+DESCRIPTION="Qt bindings for libpulse"
+HOMEPAGE="https://invent.kde.org/libraries/pulseaudio-qt"
+
+if [[ ${KDE_BUILD_TYPE} = release ]]; then
+	SRC_URI="mirror://kde/stable/${MY_PN}/${MY_P}.tar.xz"
+	KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv ~x86"
+fi
+
+LICENSE="LGPL-2.1"
+SLOT="0/3"
+
+RDEPEND="
+	>=dev-qt/qtgui-${QTMIN}:6
```

Comment:
See https://invent.kde.org/libraries/pulseaudio-qt/-/blob/master/CMakeLists.txt?ref_type=heads#L40 vs https://wiki.gentoo.org/wiki/Project:Qt/Qt6_migration_notes.

Reply:
Thanks. I didn't know that page existed.

Reply:
It's new :D

Reply:
Regardless, you would have noticed upon pkgcheck, which is a bare minimum before filing a PR.